c++/src/kj/compat/http.c++

```diff
@@ -3295,8 +3295,9 @@ public:
     if (!firstRequest) {
       // For requests after the first, require that the first byte arrive before the pipeline
       // timeout, otherwise treat it like the connection was simply closed.
-      firstByte = firstByte.exclusiveJoin(
-          server.timer.afterDelay(server.settings.pipelineTimeout)
+      firstByte = firstByte
+          .exclusiveJoin(server.timer.afterDelay(server.settings.pipelineTimeout)
```

Comment:
I would have expected a second parenthesis after ...headerTimeout). (Maybe a nitpick, as it doesn't really matter what the topology of the exclusive-join graph looks like.)

Reply:
Actually it does matter here, because there's also a .then() which specifically applies to the join of the timeout and onDrain, but not firstByte.

Reply:
That said I agree the indentation is confusing and could be improved.